configure.ac

```diff
@@ -45,9 +49,12 @@ AS_IF([test "$P4_PREFIX" = ""], [
   AC_SUBST([P4_PREFIX])
 ])
 
-diff_w_test=$($PYTHON $srcdir/tools/diff.py $P4_PATH $srcdir/bmv2/tests/test.p4)
-AM_CONDITIONAL([BMV2_TEST_ENV],
-  [test "$P4_NAME" = "test" && test "$P4_PREFIX" = "test" && test "$diff_w_test" = "no"])
+diff_w_test=$($PYTHON $srcdir/tools/diff.py $P4_PATH $srcdir/bmv2/tests/pdtest/pdtest.p4)
+
+if test "$diff_w_test" = "yes" ; then
+    diff_w_test=$($PYTHON $srcdir/tools/diff.py $P4_PATH $srcdir/bmv2/tests/openflow/oftest.p4)
+fi
+AM_CONDITIONAL([BMV2_TEST_ENV], [test "$P4_NAME" = "pdtest" || test "$P4_NAME" = "oftest" && test "$P4_PREFIX" = "pdtest" || test "$P4_PREFIX" = "oftest" && test "$diff_w_test" = "no"])
```

Comment:
Yes

On Wed, Jun 22, 2016 at 4:00 PM, Antonin Bas <notifications@github.com>
wrote:

> In configure.ac
> <https://github.com/p4lang/p4-build/pull/5#discussion_r68150657>:
>
> > @@ -45,9 +49,12 @@ AS_IF([test "$P4_PREFIX" = ""], [
> >    AC_SUBST([P4_PREFIX])
> >  ])
> >
> > -diff_w_test=$($PYTHON $srcdir/tools/diff.py $P4_PATH $srcdir/bmv2/tests/test.p4)
> > -AM_CONDITIONAL([BMV2_TEST_ENV],
> > -  [test "$P4_NAME" = "test" && test "$P4_PREFIX" = "test" && test "$diff_w_test" = "no"])
> > +diff_w_test=$($PYTHON $srcdir/tools/diff.py $P4_PATH $srcdir/bmv2/tests/pdtest/pdtest.p4)
> > +
> > +if test "$diff_w_test" = "yes" ; then
> > +    diff_w_test=$($PYTHON $srcdir/tools/diff.py $P4_PATH $srcdir/bmv2/tests/openflow/oftest.p4)
> > +fi
> > +AM_CONDITIONAL([BMV2_TEST_ENV], [test "$P4_NAME" = "pdtest" || test "$P4_NAME" = "oftest" && test "$P4_PREFIX" = "pdtest" || test "$P4_PREFIX" = "oftest" && test "$diff_w_test" = "no"])
>
> you mean you like what I am suggesting?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/p4lang/p4-build/pull/5/files/bc6ebbd5168c608c6ef0096cf45b26f409ba8efc#r68150657>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFgBcWA5H8Ij3V-PW-un6KqdTjYEZUcDks5qOb6ogaJpZM4Iylfb>
> .
>